packages/sui-js/src/number/index.js

```diff
@@ -0,0 +1,7 @@
+const numberWithDots = ({ value = 0 }) =>
+  `${value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ".")}`;
+
+const numberWithCommas = ({ value = 0 }) =>
+  `${value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",")}`;
```

Comment:
you are repeating code, what about a factory?

```javascript
const formatFactory = ({delimeter}) => ({value = 0}) =>
  `${value.toString().replace(/\B(?=(\d{3})+(?!\d))/g, delimeter)}`;

const numberWithCommas = formatFactory({delimeter: ','})
```

Reply:
that's cool. Thanks :)